dracut/30ignition/mnt-boot.mount

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Ignition Boot Mount
+ConditionPathExists=/dev/disk/by-label/boot
+
+# Wait for multipathd to start to prevent racing over
+# access to the raw device. This ensures that device symlinks
+# are setup properly.
+Wants=multipathd.service
+After=multipathd.service
```

Comment:
I wonder if we should add the multipathd stuff in a second commit with more information about the problem that is being solved.

Reply:
Yeah, I think splitting them up makes sense. Also, I'd think we'd only want `After=` here since the mount itself doesn't actually need `multipathd`.

Reply:
:+1: